floodviz/static/js/floodviz_onready.js

```diff
@@ -21,7 +21,8 @@ document.addEventListener('DOMContentLoaded', function (event) {
 		'rivers_data': FV.mapinfo.rivers_data,
 		'ref_data': FV.mapinfo.ref_data,
 		'site_data': FV.mapinfo.site_data,
-		'div_id': '#map'
+		'div_id': '#map',
+		'hydrograph_display_ids': FV.hydrograph_display_ids
```

Comment:
I would prefer this be called site_display_ids just to emphasize that they don't have to be associated with a hydrograph.